roles/openshift_health_checker/openshift_checks/disk_availability.py

```diff
@@ -1,9 +1,11 @@
 # pylint: disable=missing-docstring
+import os.path
+import tempfile
+
 from openshift_checks import OpenShiftCheck, OpenShiftCheckException, get_var
-from openshift_checks.mixins import NotContainerizedMixin
 
 
-class DiskAvailability(NotContainerizedMixin, OpenShiftCheck):
+class DiskAvailability(OpenShiftCheck):
```

Comment:
The docs loop back into the same reqs as for RPM install: https://docs.openshift.org/latest/install_config/install/rpm_vs_containerized.html#containerized-storage-requirements